src/encoded/static/components/static-pages/placeholders/MdSortableTable.js

```diff
@@ -596,14 +628,15 @@ function SortableTableRow(props) {
 }
 
 function SortableTableBody(props) {
-    const { data, columns, widths, fullRowWidth } = props;
+    const { data, columns, widths, fullRowWidth, maxContainerHeight } = props;
```

Comment:
Codacy found an issue: ['maxContainerHeight' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6227201)